src/configs/all.ts

```diff
@@ -88,7 +88,11 @@ export const rules = {
     // Functionality
     "await-promise": true,
     // "ban": no sensible default
-    "curly": true,
+    "curly": [
+        true,
+        "as-needed",
+        "ignore-same-line",
+    ],
```

Comment:
This change inverts how the rule checks files for all users of those presets. This is a breaking change and is probably not what you intended.